nixos/modules/tasks/auto-upgrade.nix

```diff
@@ -21,6 +21,17 @@ in {
         '';
       };
 
+      target = mkOption {
+        type = types.str;
```

Comment:
I think this option should be of type `types.enum ["boot" "switch"]` or something.

Reply:
Yep, would be great, but so far I don't know how to make taht.

Reply:
Just `type = types.enum ["boot" "switch"];` instead of the `type = types.str;` line.

Reply:
Ok done :)